src/gui/Gui_clean_exit_TEST.cc

```diff
@@ -77,9 +77,7 @@ void startBoth(const std::string &_fileName)
 }
 
 /////////////////////////////////////////////////
-/// TODO (azeey) Temporarliy disabled until
-/// https://github.com/gazebosim/gz-sim/issues/1443 is resolved
-TEST_P(GazeboDeathTest, DISABLED_CleanExit)
+TEST_P(GazeboDeathTest, CleanExit)
```

Comment:
This test is failing on all platforms 😕

Reply:
Actually, this is using nightlies of `plugin2`, but not of `gui7` yet, let me trigger some:

* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gz-gui7-debbuilder&build=614)](https://build.osrfoundation.org/job/gz-gui7-debbuilder/614/)
* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gz-gui7-debbuilder&build=615)](https://build.osrfoundation.org/job/gz-gui7-debbuilder/615/)

Reply:
Ok, now it passes on Ubuntu, but it's failing on macOS:

```
61: [ RUN      ] WorldFiles/GazeboDeathTest.CleanExit/0
61: 
61: [WARNING] /Users/jenkins/workspace/ignition_gazebo-ci-pr_any-homebrew-amd64/ign-gazebo/test/gtest_vendor/src/gtest-death-test.cc:1108:: Death tests use fork(), which is unsafe particularly in a threaded context. For this test, Google Test detected 3 threads. See https://github.com/google/googletest/blob/master/docs/advanced.md#death-tests-and-threads for more explanation and suggested solutions, especially if this is the last message you see before your test times out.
61: /Users/jenkins/workspace/ignition_gazebo-ci-pr_any-homebrew-amd64/ign-gazebo/src/gui/Gui_clean_exit_TEST.cc:97: Failure
61: Death test: startBoth(sdfFile)
61:     Result: died but not with expected exit code:
61:             Terminated by signal 11
61: Actual msg:
61: [  DEATH   ] The process has forked and you cannot use this CoreFoundation functionality safely. You MUST exec().
61: [  DEATH   ] Break on __THE_PROCESS_HAS_FORKED_AND_YOU_CANNOT_USE_THIS_COREFOUNDATION_FUNCTIONALITY___YOU_MUST_EXEC__() to debug.
61: [  DEATH   ] 
61: [  FAILED  ] WorldFiles/GazeboDeathTest.CleanExit/0, where GetParam() = "empty.sdf" (50 ms)
```

Windows is also failing, but it doesn't even find the shared libraries.

I'm going to disable the test for macOS and Windows.

Reply:
Also only enabled on Linux in Fortress, so thanks for making the change in https://github.com/gazebosim/gz-sim/pull/1649/commits/f0c2c27067a9898edbad12579a8993de41faa91f